src/components/gds-menu/gds-menu.stories.js

```diff
@@ -51,7 +51,7 @@ export const Submenus = () => html`
     </gds-menu-item-nested>
     <gds-menu-item-nested slot="item" submenu-icon="❯">
       <a slot="link" href="#second">
-        Second item
+        <gds-menu-item>Second item</gds-menu-item>
```

Comment:
@taromorimoto  what do you think about this? I feel it sort of makes sense since a submenu item should always look the same in addition + the dropdown icon.

Main reason I made the change was because this nastiness disappeared:

```diff
-  // Note that the link should have a padding all around for the focus styles
-  // to look good.
-  padding: var(--gds-menu-item-nested-a-padding, var(--spacing-xs) var(--spacing-m));
-
-  // Desktop from viewport width.
-  @include desktopOnly {
-    padding-left: var(--spacing-s);
-    padding-right: var(--spacing-s);
-  }
+
```

The issue is that the clickable area needs to have the padding for focus styling.

Reply:
At least this looks like a good idea? 😄